genesyscloud/tfexporter/tf_exporter_resource_test.go

```diff
@@ -164,7 +165,7 @@ func (r *registerTestInstance) registerTestResources() {
 	providerResources[knowledgeDocument.ResourceType] = knowledgeDocument.ResourceKnowledgeDocument()
 	providerResources["genesyscloud_knowledge_document_variation"] = knowledge.ResourceKnowledgeDocumentVariation()
 	providerResources["genesyscloud_knowledge_category"] = gcloud.ResourceKnowledgeCategory()
-	providerResources["genesyscloud_knowledge_label"] = gcloud.ResourceKnowledgeLabel()
+	providerResources["genesyscloud_knowledge_label"] = knowledgeLabel.ResourceKnowledgeLabel()
```

Comment:
This can become `providerResources[knowledgeLabel.ResourceType] = knowledgeLabel.ResourceKnowledgeLabel()`